src/sas/qtgui/MainWindow/UnitTesting/MainWindowTest.py

```diff
@@ -99,12 +98,13 @@ def testExit(self, qapp, mocker):
         """
         # Must mask sys.exit, otherwise the whole testing process stops.
         mocker.patch.object(sys, 'exit')
-        mocker.patch.object(HidableDialog, 'exec', return_value=True)
+        # mocker.patch.object(QtWidgets.QMessageBox, 'question', return_value=QtWidgets.QMessageBox.Yes)
+        mocker.patch.object(HidableDialog, 'exec', return_value=1)
 
         # Open, then close the main window
         screen_resolution = QtCore.QRect(0, 0, 640, 480)
         tmp_main = MainSasViewWindow(screen_resolution, None)
         tmp_main.close()
 
         # See that the MessageBox method got called
-        assert HidableDialog.exec.called
+        assert HidableDialog.called_once()
```

Comment:
This is throwing an error locally (Win10). Should this be `HidableDialog.exec.called_once()`?

Error message:

> AttributeError: type object 'HidableDialog' has no attribute 'called_once'

Reply:
Yes, it should.

Merged the change.